frontend/src/scenes/notebooks/Notebook/Editor.tsx

```diff
@@ -49,22 +61,18 @@ export function Editor({
             ExtensionPlaceholder.configure({
                 placeholder: placeholder,
             }),
-            FloatingMenu.configure({
-                shouldShow: ({ editor }) => {
-                    console.log('Floating extension')
-                    if (!editor) {
-                        return false
-                    }
-                    if (
-                        editor.view.hasFocus() &&
-                        editor.isEditable &&
-                        editor.isActive('paragraph') &&
-                        isCurrentNodeEmpty(editor)
-                    ) {
-                        return true
+            FloatingMenu.extend({
+                onSelectionUpdate() {
+                    updatePreviousNode()
+                },
+                onUpdate: () => {
+                    updatePreviousNode()
+                    resetSuggestions()
+                },
+                addKeyboardShortcuts() {
+                    return {
+                        Tab: () => true,
```

Comment:
This is necessary because the editor loses focus otherwise. Looking online the Prosemirror moderators are against this because it disrupts how element focus should work but it's necessary unless we choose a different key combo to accept a suggestion